src/components/grid-aware/VideoSpotlightBlock/VideoSpotlightBlock.module.css

```diff
@@ -0,0 +1,109 @@
+.bleedWrapper {
+  background-color: var(--color-black);
+  display: grid;
+  grid-template-columns: 1fr minmax(var(--grid-content-min-width), var(--grid-content-max-width)) 1fr;
+  grid-template-rows: 465px;
+  margin: auto;
+  max-width: var(--grid-bleed-max-width);
+}
+
+.bleedMainContent {
+  grid-column: 2;
+  /* Override grid item default of auto so that item contents cannot stretch the grid itself */
+  min-width: 0;
+}
+
+@media (--tablet-and-down) {
+  .bleedWrapper {
+    display: flex;
+    flex-direction: column;
+  }
+
+  .bleedBackground {
+    height: 250px;
+  }
+}
+
+.gridParent {
+  column-gap: var(--grid-column-gap);
+  display: grid;
+  grid-template-columns: repeat(var(--grid-column-count), minmax(0, 1fr));
+}
+
+/* Override grid item default of auto so that item contents cannot stretch the grid itself */
+.gridParent > * {
+  min-width: 0;
+}
+
+@media (--tablet-and-down) {
+  .gridParent {
+    display: block;
+  }
+}
+
+.textCardWrapper {
+  text-align: center;
+}
+
+@media (--laptop-and-up) {
+  .gridAreaImage {
+    align-items: center;
+    display: flex;
+    justify-content: center;
+    position: absolute;
+  }
+
+  .gridAreaCard {
+    grid-column: 8 / span 5;
+    position: relative;
```

Comment:
If you're only using this to make this element appear above the image, then you should just use `z-index: 1` here.

Alternatively, you could change the `.gridAreaImage` element to be positioned via the grid, rather than absolutely, and then it would naturally lie below the card here. However, doing that will require you to add another wrapper layer in order to center the play button over the image, since positioning the `.gridAreaImage` using the grid will result in that receiving the full width of the grid columns rather than the width of the image.

I can explain this over a call in more detail, if you'd like.

Reply:
I thought we were supposed to avoid the use of z-index just as a good practice. Like what if something from the navbar overlaps?

Reply:
Will change to use z-index right now though.

Reply:
I would argue that using `position: relative` just to control the stacking order is kind of obtuse, since that's not a typical usage of it. `z-index` is at least very explicit about what you're trying to do with it.

I was also somewhat hoping that you would use grid positioning on the image, which would do away with this other stuff.

I guess alternatively, you could set a negative z-index on the image, which would push it below card.

Reply:
It would be complex to use the grid on the .gridAreaImage because there is a slight offset in the downward direction that overlaps with the space outside its normal boundries.